src/api/serviceConfig.ts

```diff
@@ -2,6 +2,7 @@
 const serviceConfig: any = {
   dns_gd: {
     name: 'Godaddy',
+    service: 'https://api.godaddy.com',
```

Comment:
after thinking about it, I think its better if we leave this out. Could you hardcode this in `goDaddy.ts` instead of putting this here?